test-cli/src/bin/main.rs

```diff
@@ -88,6 +111,32 @@ fn main() {
     let mut rng: StdRng = SeedableRng::from_seed(seed_bytes);
 
     match config.command {
+        Command::GetQuotes {
+            deqs_uri,
+            base_token_id,
+            counter_token_id,
+            base_vol_min,
+            limit,
+        } => {
+            let ch =
+                ChannelBuilder::default_channel_builder(env).connect_to_uri(&deqs_uri, &logger);
+            let client_api = DeqsClientApiClient::new(ch);
+
+            let mut req = GetQuotesRequest {
+                base_range_min: base_vol_min,
+                base_range_max: u64::MAX,
+                limit,
+                ..Default::default()
+            };
+            req.set_pair(Pair {
+                base_token_id: *base_token_id,
+                counter_token_id: *counter_token_id,
+                ..Default::default()
+            });
+            let resp = client_api.get_quotes(&req).expect("get quotes failed");
+            println!("got successful response");
```

Comment:
Can probably skip this print

Reply:
its helpful when there's nothing in the quotebook, just to be sure that you actually got success after you just deployed the deqs